apps/api/src/app/layouts/usecases/delete-layout/delete-layout.use-case.ts

```diff
@@ -33,5 +35,25 @@ export class DeleteLayoutUseCase {
     }
 
     await this.layoutRepository.deleteLayout(command.layoutId, layout._environmentId, layout._organizationId);
+
+    const createLayoutChangeCommand = CreateLayoutChangeCommand.create({
+      environmentId: command.environmentId,
+      layoutId: command.layoutId,
+      organizationId: command.organizationId,
+      userId: command.userId,
+    });
+
+    await this.createChange(command);
+  }
+
+  private async createChange(command: DeleteLayoutCommand): Promise<void> {
```

Comment:
Why is this function needed? Doesn't it do the same thing by creating the command? 
Maybe you meant to remove the first CreateLayoutChangeCommand in line 39?